client/app/src/models/app/shared-public-model.ts

```diff
@@ -133,6 +133,8 @@ export interface Comment {
   content: string;
   author_id: string;
   visibility: string;
+  type: string;
```

Comment:
Looks like `type` in this case is better an `enum` than a `string`. This way, there'll be consistency.